tests/modules/programs/tmux/not-enabled.nix

```diff
@@ -0,0 +1,13 @@
+{ config, lib, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.tmux = { enable = false; };
+
+    nmt.script = ''
+      !assertFileExists home-files/.tmux.conf
```

Comment:
This won't work, the script is just a regular bash script. You can use `assertFileNotExists` instead. You may need to rebase on latest master first.

Reply:
Awesome, thank you for updating your test library for this!